server/services/TransferService.js

```diff
@@ -115,6 +202,31 @@ class TransferService {
 
       await this._session.commitTransaction();
 
+      if (transfer) {
```

Comment:
Also I don't think this is sufficient to determine if the transfer was successfully accepted. Also have you confirmed the contents of result, possible it might contain the values needed by the logs

Reply:
Confirmed this?

Reply:
{
  id: 'e188c693-feab-494b-96ca-01af737f766f',
  type: 'send',
  parameters: { tokens: [ '958111ed-41ef-4230-8eca-2dcd1ededde5' ] },
  state: 'completed',
  created_at: 2023-09-30T06:51:38.644Z,
  closed_at: 2023-09-30T06:51:38.644Z,
  active: true,
  claim: false,
  originating_wallet: 'walletA',
  source_wallet: 'walletA',
  destination_wallet: 'walletB'
}

this is the results returned by the transfer. I will be using this in the payload for the eventLog

Reply:
confirmed at the top and changed all the payload where only transferId is saved to result